File: JS Essentials/3.0 Conditional Statements/08. Toy Shop.js
function toyShop(arg) {
    let [trip, puzzle, doll, bear, minion, truck] = arg.map(Number)
    let toysCount = puzzle + doll + bear + minion + truck;
    let toysTotal = (puzzle * 2.60) + (doll * 3) + (bear * 4.1) + (minion * 8.20) + (truck * 2);

    if (toysCount >= 50) toysTotal = toysTotal * 0.75
    toysTotal = toysTotal * 0.90

    if (toysTotal >= trip)
        return `Yes! ${(toysTotal - trip).toFixed(2)} lv left.`
    return `Not enough money! ${(trip - toysTotal).toFixed(2)} lv needed.`
}

// console.log(toyShop([40.8, 20, 25, 30, 50, 10, ]))

File: Python Essentials/3.0 Conditional Statements/08. Toy Shop.py
trip, puzzle, doll, bear, minion, truck = [float(input()) for _ in range(6)]
toys_count = puzzle + doll + bear + minion + truck
toys_total = (puzzle * 2.60) + (doll * 3) + (bear * 4.1) + (minion * 8.20) + (truck * 2)

if toys_count >= 50:
    toys_total *=  0.75
toys_total *= 0.90

if toys_total >= trip:
    print(f'Yes! {(toys_total - trip):.2f} lv left.')
else:
    print(f'Not enough money! {(trip - toys_total):.2f} lv needed.')

